packages/app-desktop/gui/MenuBar.tsx

```diff
@@ -421,8 +422,31 @@ function useMenu(props: Props) {
 
 			function _showAbout() {
 				const v = versionInfo(packageInfo);
+				const pluginService = PluginService.instance().plugins;
+				function dialogInfo(pluginService: any): string {
+					let content = v.message.concat('\n');
+					const items = [];
+					for (const plugins in pluginService) {
+						const plugin = pluginService[plugins];
+						items.push({
+							name: plugin.manifest.name,
+							version: plugin.manifest.version,
+						});
+						items.sort((a: any, b: any) => {
+							return a.name < b.name ? -1 : +1;
+						});
+					}
+					if (items) {
+						for (const item of items) {
+							content = content.concat(`\n${item.name}: ${item.version}`);
+						}
+					}
+
+					return content;
+				}
```

Comment:
Honestly this code is a joke. I don't even know things like `pluginService[plugins]` can work and all you are doing is mapping a structure to the same structure, so you're doing nothing really. To top it up you're sorting the array within the loop and check if the "items" variable is set when it obviously always is.

This is a project used by many people, not a place for your to learn how to code. So I'm closing this, and please do not open a new pull request.